pkg/config/config.go

```diff
@@ -36,3 +40,13 @@ func GetMaxWaitPerMachine() time.Duration {
 	}
 	return DefaultMaxWaitPerMachine
 }
+
+func GetExternalEtcdTimeout() string {
+	if env, found := os.LookupEnv(ExternalEtcdTimeoutEnv); found {
+		if _, err := time.ParseDuration(env); err == nil {
+			return env
+		}
+		logger.V(3).Info(fmt.Sprintf("Invalid EKSA_EXTERNAL_ETCD_TIMEOUT value: %s Use the default timeout: %s", env, ExternalEtcdTimeoutEnv))
```

Comment:
I know we didn't do this for the others, but can we use the constant defined above here?